src/config/passport.js

```diff
@@ -0,0 +1,45 @@
+import passport from 'passport';
+import {
+  Strategy as GoogleStrategy
+} from 'passport-google-oauth20';
+import env from './env-config';
+import Helpers from '../utils/helpers';
+
+const FacebookStrategy = require('passport-facebook').Strategy;
+
+const {
+  passportCb
+} = Helpers;
+
+const {
+  GOOGLE_CLIENT_ID,
+  GOOGLE_CLIENT_SECRET,
+  FACEBOOK_CLIENT_ID,
+  FACEBOOK_CLIENT_SECRET
+} = env;
+
```

Comment:
These guys, line 15-18 is also not necessary. use env variables like this `process.env.GOOGLE_CLIENT_ID`